### PR TITLE
Add tests for GameRound/(Elimination)Scoreboard/(Elimination)Player

### PR DESCRIPTION
**Pull request by Brainicism**

### CODE REVIEW

File: src/helpers/discord_utils.ts [hunk 1]
```diff
@@ -131,8 +131,8 @@ export async function sendEndOfRoundMessage(messageContext: MessageContext, scor
         text: "",
     };
 
-    if (gameRound.songAliases.length > 0) {
-        footer.text += `Aliases: ${Array.from(gameRound.songAliases).join(", ")}\n`;
+    if (gameRound.acceptedSongAnswers.length > 1) {
+        footer.text += `Aliases: ${Array.from(gameRound.acceptedSongAnswers).join(", ")}\n`;
```
Comment (taahamahdi) [inline src/helpers/discord_utils.ts:135]: Will we be showing the song name here too then (when another alias exists)?

**Review by taahamahdi — APPROVED** [review]

File: src/helpers/discord_utils.ts (hunk 1 above)
Comment (Brainicism) [reply src/helpers/discord_utils.ts:135]: Yes but I don't think that's a major issue, being able to see every accepted answer

File: src/test/elimination_scoreboard.test.ts [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        this.eliminationScoreboard.addPlayer(userIds[1], "seulgi#7854", "someurl");
+        this.eliminationScoreboard.addPlayer(userIds[2], "joy#4144", "someurl");
+    });
+    describe("nobody has a score yet", function () {
+        it("should return an empty array", function () {
+            assert.deepStrictEqual(this.eliminationScoreboard.getWinners(), []);
+        });
+    });
+
+    describe("single player, has guessed atleast once", function () {
```
Comment (taahamahdi) [inline src/test/elimination_scoreboard.test.ts:68]: at least

File: src/test/elimination_scoreboard.test.ts [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        it("should return the weakest person's number of lives", function () {
+            this.eliminationScoreboard.addPlayer(userIds[0], "irene#1234", "someurl", 5);
+            this.eliminationScoreboard.addPlayer(userIds[1], "seulgi#7854", "someurl", 8);
+            this.eliminationScoreboard.addPlayer(userIds[2], "joy#4144", "someurl", 2);
+            assert.strictEqual(this.eliminationScoreboard.getLivesOfWeakestPlayer(), 2);
+        });
+    });
+    describe("tie for the weakest", function () {
+        it("should return the number of lives", function () {
+            this.eliminationScoreboard.addPlayer(userIds[0], "irene#1234", "someurl", 2);
```
Comment (taahamahdi) [inline src/test/elimination_scoreboard.test.ts:147]: Might benefit making one of these greater than 2

File: src/test/elimination_scoreboard.test.ts [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+            assert.strictEqual(this.eliminationScoreboard.getLivesOfWeakestPlayer(), 2);
+        });
+    });
+});
+
+describe("starting lives", function () {
+    describe("no explicit number of lives set for player", function () {
+        it("should default to the scoreboard's default", function () {
+            this.eliminationScoreboard.addPlayer(userIds[0], "irene#1234", "someurl");
+            assert.strictEqual(this.eliminationScoreboard.getPlayerLives(userIds[0]), 10);
```
Comment (taahamahdi) [inline src/test/elimination_scoreboard.test.ts:159]: Use the constant DEFAULT_LIVES

**Review by taahamahdi — CHANGES_REQUESTED** [review]

**Review by taahamahdi — APPROVED** [review]

**Pull request merged**